grammars/tree-sitter-typescript.cson

```diff
@@ -0,0 +1,110 @@
+id: 'typescript'
+name: 'TypeScript'
+type: 'tree-sitter'
+parser: 'tree-sitter-typescript'
+
+fileTypes: ["ts", "tsx", "js.flow"]
```

Comment:
Confused as to why js.flow is in the typescript grammar.

Reply:
Oh, thanks for catching that. Originally, I was going to just have one grammar for both.